### ISSUE TITLE
Runtime is not ready for debugging

### ISSUE THREAD

**Issue by Syahrul** [jhen0409/react-native-debugger#390]
React Native Debugger app version: 0.9.10
React Native version: EXPO SDK 34
Platform: IOS
Is real device of platform: No
Operating System: macOS High Sierra

Everytime I open React Native Debugger, it works for a few seconds and then I got a red screen showing "Runtime is not ready for debugging. Make sure Packager server is running"

Not sure why, the app runs okay if I turned off React Native Debugging. I don't have any other alternative since react-devtools is not working anymore. I need to inspect the element and fix the layout.

![Simulator Screen Shot - iPhone 5s - 2019-08-21 at 16 37 06](https://user-images.githubusercontent.com/2231998/63416491-10285b80-c432-11e9-8554-e2e205a79115.png)

Thanks

**Comment by will-stone** [comment]
Hi, Did you manage to find a solution to this issue?

**Comment by PrimusLab** [comment]
No, I haven't find a solution yet. Didn't use it since I post this issue.

**Comment by will-stone** [comment]
A new version got released today, I'm gonna see if that fixes the issue. Won't know until I go to work tomorrow, but will update with the results.

**Comment by PrimusLab** [comment]
Looks like the error is gone, I was able to debug the layout with no error thrown. Hopefully yours is the same.

**Comment by kevinvugts** [comment]
@PrimusLab are you able to debug ios app properply? I am still experiencing big delays.

**Comment by will-stone** [comment]
It's still not working for me. Could this be to do with macOS? I feel I only started getting this after upgrading to 10.14.6.

**Comment by will-stone** [comment]
Restarted my mac and all is working now. I guessing the OS update didn't reset something correctly.

**Comment by kevinvugts** [comment]
Mine doesn't work still. Still buggy and really slow af.

**Comment by okeybond** [comment]
Also happening on Mojave, expo 35

**Comment by travislang** [comment]
I am having the same issue. It was working fine up until today. 

I was able to isolate the problem to happen only when I instantiate a new socket.io websocket connection within my app. 

Coincidentally I just upgraded Xcode to Version 11 and have had other issues getting an expo app to start. I have to manually ever the host into the expo client on simulator. 

I wonder if it has to do with multiple websocket connections going through a chrome webworker? The packager and the websocket connection within my app...?

**Comment by jacklj** [comment]
I'm having the same issue - latest version of react-native-debugger still doesn't work.

**Comment by vrgimael** [comment]
Yeah, this is still pretty annoying and I get it totally randomly

**Comment by ramon90** [comment]
same issue, it has been there for years!!!!

**Comment by garrett-gottlieb** [comment]
I'm getting this whenever I...
1. Run react-devtools or react-native-debugger
2. CMD + D > Show Inspector
3. Tap on anything

I am using a yarn workspace monorepo. Here's my `react-native info`:
```
System:
    OS: macOS 10.15.4
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 1.12 GB / 16.00 GB
    Shell: 3.0.2 - /usr/local/bin/fish
  Binaries:
    Node: 12.16.2 - /var/folders/_5/jx_v38nn2yg5g__c9lp_l2nw0000gn/T/yarn--1590257718031-0.5595627091006565/node
    Yarn: 1.22.4 - /var/folders/_5/jx_v38nn2yg5g__c9lp_l2nw0000gn/T/yarn--1590257718031-0.5595627091006565/yarn
    npm: 6.14.4 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27
      Build Tools: 19.1.0, 21.1.2, 23.0.1, 23.0.2, 23.0.3, 24.0.3, 25.0.0, 25.0.2, 26.0.2, 26.0.3, 28.0.2, 28.0.3
      System Images: android-29 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  npmPackages:
    react: ^16.13.1 => 16.13.1
    react-native: ~0.61.4 => 0.61.5
  npmGlobalPackages:
    react-native-git-upgrade: 0.2.7
```

**Comment by gino8080** [comment]
any news??

**Comment by devarif** [comment]
any solution for this please

**Comment by FrankieJLyons** [comment]
Any ideas people?

**Comment by moparlakci** [comment]
solved the issue by running expo start, or npm start by a normal user, not sudo 

Hope it helps anyone
